### DISCUSSION TITLE
How can I make my app compatible with Chrome 75?

### DISCUSSION THREAD

**Discussion by kaikaibenkai** [nuxt/nuxt#21136] [Questions]
Chrome 75 appears to not support `<script module>`, so my app shows a blank page...

<details>
<summary>I tried `@vitejs/plugin-legacy`,  but it only slowed down the startup speed of DevServer.</summary>

```ts
import legacy from '@vitejs/plugin-legacy';

export default defineNuxtConfig({
  ssr: false,
  modules: [
    '@vueuse/nuxt',
  ],
  vite: {
    plugins: [
      legacy({
        targets: {
          chrome: '74',
        },
      }),
    ],
  },
  devServer: {
    port: 5703,
  },
});
```

</details>

How can I make it work in such a legacy browser? Thanks a lot!

**Comment by maou-shonen** [comment]
Do you mean the development environment?
you should be able to use webpack

**Reply by kaikaibenkai** [reply]
Dev Server is always the latest version of Chrome, but production environment needs to support Chrome 75.

**Reply by kaikaibenkai** [reply]
Could Vite do it 😥 I really can't stand the long HMR time and complicated config of webpack

**Reply by maou-shonen** [reply]
Nuxt 3 uses esbuild as the default bundler. I'm not very familiar with it myself, but I believe you should be able to find what you need in the [esbuild documentation](https://esbuild.github.io/getting-started/#bundling-for-the-browser).

I recall that esbuild also references tsconfig.json.
You can directly modify tsconfig.json, or you can modify nuxt.config.ts or package.json.

**Comment by rafifn** [comment]
it works for me [https://github.com/nuxt/nuxt/issues/15464#issuecomment-1469587401]()
